climpred/metrics.py

```diff
@@ -223,6 +290,7 @@ def _pearson_r(forecast, verif, dim=None, **metric_kwargs):
         * climpred.pearson_r_eff_p_value
     """
     metric_kwargs = _sanitize_kwargs(metric_kwargs)
+    dim = _rename_dim(dim, forecast, verif)
```

Comment:
well actually thats not ideal because we have to think about this every time when implementing a new metric and also in the custom metrics. should be handled before actually. this will require a further cleanup PR in the near future, which I can do.

Reply:
handled by https://github.com/pangeo-data/climpred/issues/446